src/dev/lupio/lupio_tmr.cc

```diff
@@ -51,15 +51,9 @@ LupioTMR::LupioTMR(const Params &params) :
     nThread(params.num_threads),
     tmrEvent([this]{ lupioTMRCallback(); }, name()),
     startCycle(0)
-{
-    start = curTick(); 
```

Comment:
If variable `start` becomes unused entirely, you'd probably want to remove it from the class definition in the header file.